jax_unirep/utils.py

```diff
@@ -119,9 +126,20 @@ def aa_seq_to_int(s: str) -> List[int]:
     return [24] + [aa_to_int[a] for a in s] + [25]
 
 
-def load_embedding(folderpath: Optional[str] = None):
-    """Load pre-trained embedding weights for UniRep1900 model."""
-    weights_dir = get_weights_dir(folderpath=folderpath)
+def load_embedding(
+    folderpath: Optional[str] = None, paper_weights: Optional[int] = 1900
+):
+    """
+    Load pre-trained embedding weights for UniRep paper models.
+
+    :param folderpath: Path to the folder containing the model weights
+    :param paper_weights: If paper weights should be loaded (folderpath set to None),
+        specify from which model architecture. Possible values are 1900, 256 and 64.
```

Comment:
Wonderful docs. Only small nitpick:

```suggestion
        specify from which model architecture. Possible values are `1900`, `256` and `64`.
```